Update pubsub.md (#2329)
redis/redis-doc: docs/manual/pubsub.md

message. The commands that are allowed in the context of a subscribed
client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET`, and `QUIT`.

> In particular, the operation of the UNSUBSCRIBE and PUNSUBSCRIBE commands in redis-cli has limitations, so keep in mind.

Reviewer: This will generate a hyperlink for the command
```suggestion
In particular, the operation of the `UNSUBSCRIBE` and `PUNSUBSCRIBE` commands in redis-cli has limitations, so keep in mind.
```

> In particular, the operation of the `UNSUBSCRIBE` and `PUNSUBSCRIBE` commands in redis-cli has limitations, so keep in mind.

Reviewer 2: I think it would be clearer to combine this statement with the next statement regarding redis-cli pub/sub limitations.
maybe something like:
```
Please note that `redis-cli` will not accept any commands once in
subscribed mode including the allowed commands mentioned earlier.
In particular, you will not be able to use the `UNSUBSCRIBE` and `PUNSUBSCRIBE`  commands 
via redis-cli once in subscribed mode and can only quit the mode with `Ctrl-C`.
```

Author: @ranshid, Thank you for your review.
Yes, I think it's good to mention it in a combined sentence. :)

Please note that `redis-cli` will not accept any commands once in

> subscribed mode and can only quit the mode with `Ctrl-C`.

Author: @zuiderkwast  Thanks again for your review.

I combined the sentences as below and created two candidates. 
Please confirm. :)

(1)
If the client using redis-cli is in subscribed mode, redis-cli will not accept the commands once and can only quit the mode with Ctrl-C. In particular, the commands such as `UNSUBSCRIBE` and `PUNSUBSCRIBE` cannot be used.

or

(2)
When a client using redis-cli is in subscription mode, commands such as `UNSUBSCRIBE` and `PUNSUBSCRIBE` cannot be used because redis-cli will not accept the commands once more and can only quit the mode with Ctrl-C.

> subscribed mode and can only quit the mode with `Ctrl-C`.

Reviewer 3: Both are good, but maybe start with "Please note" as before.

> subscribed mode and can only quit the mode with `Ctrl-C`.

Author: @zuiderkwast  Thanks for checking.

Please note that when a client using redis-cli is in subscription mode, commands such as 'UNSUBSCRIBE' and 'PUNSUBSCRIBE' cannot be used because redis-cli will not accept the commands once more and can only quit the mode with Ctrl-C.

message. The commands that are allowed in the context of a subscribed
client are `SUBSCRIBE`, `SSUBSCRIBE`, `SUNSUBSCRIBE`, `PSUBSCRIBE`, `UNSUBSCRIBE`, `PUNSUBSCRIBE`, `PING`, `RESET`, and `QUIT`.

> Please note that when a client using `redis-cli` is in subscription mode, 

Reviewer 3: Sorry, there is still something in this sentence that seems odd.

* "a client using redis-cli" is strange because redis-cli is a client.
* The grammar "when using redis-cli is in ..." seems wrong. Better without "is" and with a comma instead.

```suggestion
Please note that when using `redis-cli`, in subscribed mode
```

Author: In sentences, I think it's better without "is" and with a ',' comma instead.
And I agree to exclude 'a client' as well.
@zuiderkwast, Thank you for your detailed review. :)

commands such as `UNSUBSCRIBE` and `PUNSUBSCRIBE` cannot be used because 

> `redis-cli` will not accept the commands once more and can only quit the mode with `Ctrl-C`.

Reviewer 3: Actually redis-cli doesn't accept any commands at all in this mode. Maybe even more clear is:

```suggestion
`redis-cli` will not accept any commands and can only quit the mode with `Ctrl-C`.
```

Author: Okay! now it seems clear.
